source/common/http/codec_helper.h

```diff
@@ -7,6 +7,18 @@ namespace Http {
 
 class StreamCallbackHelper {
 public:
+  void runLowWatermarkCallbacks() {
+    for (StreamCallbacks* callbacks : callbacks_) {
+      callbacks->onBelowWriteBufferLowWatermark();
```

Comment:
Oh wait, do you mean under the stack of a callback or just when the stream is in the high watermark state?